deps/wazuh_testing/wazuh_testing/dashboard_saturation_tests/data/dashboard_saturation_tests.py

```diff
@@ -0,0 +1,388 @@
+# Copyright (C) 2024, Wazuh Inc.
+# Created by Wazuh, Inc. <info@wazuh.com>.
+# This program is a free software; you can redistribute it and/or
+# modify it under the terms of GPLv2
+
+"""Dashboard Saturation Tests Module
+
+This script allows you to run dashboard stress tests. These tests
+are performed with Artillery and Playwright. Artillery and
+Playwright must be installed for it to work properly.
+"""
+
+import argparse
+import json
+import time
+
+from argparse import Namespace
+from datetime import datetime
+from os import getcwd, makedirs
+from os.path import isabs, join
+from subprocess import run
+
+import pandas as pd
+import yaml
```

Comment:
Impots shoud follow pep8: https://peps.python.org/pep-0008/#imports

> Imports should be grouped in the following order:
> 
>     Standard library imports.
>     Related third party imports.
>     Local application/library specific imports.
> 
> You should put a blank line between each group of imports.

No extra white line is necessary between time and argpase imports.
In addition remove the import of the Namespace importing the argparse directly or import along with the Namespace the ArgumentParsers:
```python
import argparse
```

```suggestion
import json
import time
from argparse import Namespace
from datetime import datetime
from os import getcwd, makedirs
from os.path import isabs, join
from subprocess import run

import pandas as pd
import yaml
```

Reply:
Solved in https://github.com/wazuh/wazuh-qa/pull/5629/commits/56f5d2f0bf8080dbe3bf7f1f58d5c7b3420d84f3